src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngine.cs

```diff
@@ -78,11 +78,29 @@ public class RazorViewEngine : IRazorViewEngine
             _pageActivator = pageActivator;
             _htmlEncoder = htmlEncoder;
             _logger = loggerFactory.CreateLogger<RazorViewEngine>();
-            _razorProject = razorProject;
+            _razorFileSystem = razorProject;
             _diagnosticSource = diagnosticSource;
             ViewLookupCache = new MemoryCache(new MemoryCacheOptions());
         }
 
+        /// <summary>
+        /// Initializes a new instance of the RazorViewEngine
+        /// </summary>
+        public RazorViewEngine(
```

Comment:
The old constructor should be marked obsolete, and add it as a thing to clean up in 3.0